Zeeguu Reader/Zeeguu Reader/ArticleViewController.swift

```diff
@@ -320,13 +320,18 @@ class ArticleViewController: UIViewController, WKNavigationDelegate, WKScriptMes
 			action.setBookmarkID(b)
 			
 			if (self.pronounceTranslatedWord) {
+				let _ = try? AVAudioSession.sharedInstance().setCategory(AVAudioSessionCategoryPlayback)
+				let _ = try? AVAudioSession.sharedInstance().setActive(true)
+				
 				let synthesizer = AVSpeechSynthesizer()
 				
 				let utterance = AVSpeechUtterance(string: word)
 				utterance.voice = AVSpeechSynthesisVoice(language: self.article?.feed.language)
 				
 				synthesizer.stopSpeakingAtBoundary(AVSpeechBoundary.Immediate)
 				synthesizer.speakUtterance(utterance)
+				
+				let _ = try? AVAudioSession.sharedInstance().setActive(false)
```

Comment:
I find this let _ = try? ... syntax interesting. is it better to have three of those instead of wrapping the entire block into a single try?